pkg/logs/sources/go.mod

```diff
@@ -101,3 +101,9 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/DataDog/datadog-agent/pkg/api => ../../api
```

Comment:
Should these replaces be grouped in with the existing replaces at the top of the file?

Reply:
This replace where added automatically to avoid the pain of having to add them manually. We used `go mod edit -replace ...` The tool does not group them together. So I think we could do that manually, at the same time since the addition of replace directive is done automatically we do not have to worry about grouping them as it has the same effect as not grouping them 😄